rotkehlchen/tasks/calendar.py

```diff
@@ -460,7 +460,7 @@ def maybe_create_l2_bridging_reminder(self) -> None:
                         event=bridge_event,
                         color=BRIDGE_CALENDAR_COLOR,
                         counterparty=bridge_event.counterparty,  # type: ignore[arg-type]  # counterparty is always present
-                        timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS)),  # noqa: E501
+                        timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS * 1000)),  # noqa: E501
```

Comment:
This WEEK_IN_SECONDS * 1000 is used a lot in this file. We should absttract it to WEEK_IN_MS and not recalculate it each time.